### PR TITLE
LPS-168385 Add back title to back button

### PR DESCRIPTION
**Pull request by veroglez**
https://issues.liferay.com/browse/LPS-168385

**Case 1:**

1. Applications Menu > Users and Organizations > User > Add New User > Add Custom Field.
2. This back redirects you to Add User (see the title).

**Case 2:**

1. Applications Menu > Users and Organizations > User > Add New User > Add Custom Field.
2. In the breadcrumb, go to View Attributes.
3. Click the Add Custom Field button.
4. This back redirects you to User (see the title).

### CODE REVIEW

File: modules/apps/expando/expando-web/src/main/java/com/liferay/expando/web/internal/display/context/ExpandoDisplayContext.java [hunk 1]
```diff
@@ -96,7 +96,10 @@ public CreationMenu getCreationMenu() throws PortalException {
 					liferayPortletResponse.createRenderURL(), "mvcPath",
 					"/edit/select_field_type.jsp", "redirect",
 					PortalUtil.getCurrentURL(_httpServletRequest),
-					"modelResource", modelResource);
+					"modelResource", modelResource, "backTitle",
+					LanguageUtil.format(
+						_httpServletRequest, "go-to-x",
+						LanguageUtil.get(_httpServletRequest, "user"), false));
```
Comment (drewbrokke) [inline modules/apps/expando/expando-web/src/main/java/com/liferay/expando/web/internal/display/context/ExpandoDisplayContext.java:102]: This is not guaranteed to always be related to a `User`. This UI is also accessible via the Custom Fields portlet under the app menu `Control Panel > Custom Fields`, and could relate to any of the model types that support custom fields.

Comment (drewbrokke) [reply modules/apps/expando/expando-web/src/main/java/com/liferay/expando/web/internal/display/context/ExpandoDisplayContext.java:102]: The `modelResource` variable here should give the correct information as to which type of model's fields are being edited.

**Review by drewbrokke — CHANGES_REQUESTED** [review]

File: modules/apps/expando/expando-web/src/main/java/com/liferay/expando/web/internal/display/context/ExpandoDisplayContext.java (hunk 1 above)
Comment (veroglez) [reply modules/apps/expando/expando-web/src/main/java/com/liferay/expando/web/internal/display/context/ExpandoDisplayContext.java]: Oh thanks for this! I update the code obtaining the information with `ResourceActionsUtil.getModelResource`

**Review by drewbrokke — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:forward

**Comment by drewbrokke** [comment]
@patriciajeanperez are these failures related?

**Comment by patriciajeanperez** [comment]
No related failures, @drewbrokke

**Comment by patriciajeanperez** [comment]
ci:forward:force

**Comment by patriciajeanperez** [comment]
ci:forward:force

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**